drivers/input/input_touch.c

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2022 Antmicro <www.antmicro.com>
```

Comment:
Should year here be 2022, or was this a copy paste mistake?

Reply:
Good catch, should be 2024 now